### PR TITLE
109 lobby images

### PR DESCRIPTION
**Pull request by DeNic0la**
This has still a lot of Security weaknesses and UI Imporvments to do, but we need to Delete all Lobbys after this so maybe do it now

### CODE REVIEW

File: functions/src/index.ts [hunk 1]
```diff
@@ -24,7 +24,11 @@ exports.lobby = functions
         const uid = idToken?.uid || '';
 
         const user = await userService.getUser(uid);
-        const lobby = await lobbyService.createLobby(user, req.body.topic);
+        const lobby = await lobbyService.createLobby(
```
Comment (Sili-Dev) [inline functions/src/index.ts:27]: add on line 16 `!req.body.req.body.topic && !req.body.imgUrl`

File: src/app/storify/storify-explore/storify-explore.component.ts [hunk 2]
```diff
@@ -1,41 +1,105 @@
...
+              next: (value) => {
+                console.log(value);
+              },
+              complete: () => {
+                console.groupEnd();
+                this.ref?.getDownloadURL().subscribe((value) => {
+                  this.createSeLobby(topic, value);
+                  console.log(value);
+                });
+                console.log('POW');
```
Comment (Sili-Dev) [inline src/app/storify/storify-explore/storify-explore.component.ts:86]: remove log

File: src/app/storify/storify-explore/storify-explore.component.ts [hunk 3]
```diff
@@ -1,41 +1,105 @@
...
-        this.router.navigate(['/storify/lobby/', value]);
-      });
+      if (
+        this.input &&
+        this.input.nativeElement.files &&
+        this.input.nativeElement.files &&
+        this.input.nativeElement.files.length > 0
+      ) {
+        let file = this.input.nativeElement.files.item(0);
+        console.log('Has file');
```
Comment (Sili-Dev) [inline src/app/storify/storify-explore/storify-explore.component.ts:66]: remove log, etc.

**Review by Sili-Dev — CHANGES_REQUESTED** [review]

File: src/app/storify/storify-explore/storify-explore.component.ts (hunk 2 above)
Comment (DeNic0la) [reply src/app/storify/storify-explore/storify-explore.component.ts]: done

File: functions/src/index.ts (hunk 1 above)
Comment (DeNic0la) [reply functions/src/index.ts:27]: done

**Review by Sili-Dev — DISMISSED** [review]

**Review by Sili-Dev — APPROVED** [review]

**Pull request merged**